wbooks/src/app/components/CardBook/index.tsx

```diff
@@ -11,19 +13,25 @@ interface Props {
   id: number;
 }
 
-export default function CardBook({ image, title, author }: Props): ReactElement {
+export default function CardBook({ image, title, author, id }: Props): ReactElement {
+  const navigation = useNavigation();
+  const handleClick = () => {
+    navigation.navigate('DetailBook', { image, title, author, id });
```

Comment:
you can have a routes.ts file that contains all the Routes names from screens. U can import it then and use it here.